src/inverseRobot.js

```diff
@@ -7,7 +7,15 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const arr = [];
+
+  for (const i of Object.entries(robot)) {
+    arr.push(i.reverse());
+  };
+
+  const res = Object.fromEntries(arr);
+
+  return Object.keys(res).length < Object.values(robot).length ? null : res;
```

Comment:
It looks too complicated. Consider making it easier.
Just iterate through robot's keys and create a reversed property in a new robot. 
If it's already present - return null, that's all